src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_UsingStatement.cs

```diff
@@ -395,5 +361,46 @@ private BoundStatement RewriteUsingStatementTryFinally(SyntaxNode syntax, BoundB
 
             return tryFinally;
         }
+
+        private BoundExpression GenerateDisposeCall(SyntaxNode syntax, BoundExpression disposedExpression, SyntaxToken awaitKeywordOpt, AwaitableInfo awaitOpt, MethodSymbol methodOpt)
```

Comment:
Why label `awaitKeywordOpt` with the `Opt` suffix when it's a struct? If it were truly optional I'd expect it to have `= default` here.

Reply:
Renamed